README.md

```diff
@@ -21,6 +21,9 @@
 - Use database migration fails to maintain database schema.
```

Comment:
### Optional
- [ ] It would be better if we add validations to our model so that users can not submit empty data which could be costly for database management. Also try and add a button to go back instead of manually navigating back and forth

Reply:
Thank you, I'm going to add it!!